force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html

```diff
@@ -1,20 +1,22 @@
 <template>
     <c-modal
         header={labels.newSchedule}
-        ondialogclose={handleFinish}
+        ondialogclose={handleClose}
         default-visible="true"
         size="large"
     >
-        <template if:true={showStep1}>
-            <c-new-service-schedule service-id={serviceId}></c-new-service-schedule>
+        <template if:true={isStep1}>
+            <c-new-service-schedule
+                service-schedule-model={serviceScheduleModel}
+            ></c-new-service-schedule>
         </template>
-        <template if:true={showStep2}>
+        <template if:true={isStep2}>
             <c-review-sessions></c-review-sessions>
         </template>
         <template if:true={showStep3}>
```

Comment:
should this be `isStep3` ?

Reply:
Nice catch, this was from the merge and I didn't catch it!